src/main/java/com/box/sdk/BoxMetadataCascadePolicy.java

```diff
@@ -0,0 +1,251 @@
+package com.box.sdk;
+
+import java.net.URL;
+
+import com.eclipsesource.json.JsonObject;
+import com.eclipsesource.json.JsonValue;
+
+/**
+ * Represents a Metadata Cascade Policy.
+ */
+public class BoxMetadataCascadePolicy extends BoxResource {
```

Comment:
Since we're waiting to ship until the API has a `type` field, you can probably just put the resource decorator back here and use the expected value of `"metadata_cascade_policy"`